Makefile

```diff
@@ -298,6 +298,29 @@ release-windows: clean all
 		CHANGELOG.md \
 		teleport/
 	mv teleport/tsh teleport/tsh.exe
+	# TODO: remove this
+	@WINDOWS_SIGNING_CERTIFICATE=$$(cat cert-dummy.pfx | base64); \
+	if [ ! -z "$${WINDOWS_SIGNING_CERTIFICATE}" ]; then \
```

Comment:
I would be tempted to make this target fail safe and exit with error if the Windows signing certificate is not provided, so that we can't ever accidentally release an unsigned `tsh.exe` due to an empty/missing secret or a lack of access, etc. To make local development easier, this could potentially be gated to only exit when running in CI by checking whether the `DRONE` environment variable is set, similar to this: https://github.com/gravitational/teleport/blob/master/build.assets/Makefile#L49

Another alternative would be to do a similar thing to what we do with RPMs - have a `windows-unsigned` target which doesn't use a signature (for local development builds etc): https://github.com/gravitational/teleport/blob/master/Makefile#L687-L690

Reply:
I'd go even further and make signing a seperate make target (with Gus's fail safe property) that depends on the unsigned artifact.

Reply:
I've now split this into `release-windows-unsigned` (which generates the usual .zip file with a `-unsigned` suffix) and `release-windows` which extracts the original .zip, signs + renames the exe, and rezips everything.